pages/intro.js

```diff
@@ -0,0 +1,224 @@
+        .doc('IntroTestHackathon')
+        .update(updateObj);
+      setWebsiteData({
+        ...websiteData,
+        [type]: {
+          ...websiteData[type],
+          header,
+          content,
+          time,
+          editor: currUserName
+        }
+      });
+    } catch (error) {
+      console.error(error);
+      alert('An error occured while trying to update the text section');
+    } finally {
+      setIsEditingObj({
+        ...isEditingObj,
+        [type]: false
+      });
+    }
+  };
+
+  const handleEditChange = (event, type, isHeader) => {
+    const newVal = event.target.value;
+    const updatedVal = isHeader
+      ? { header: newVal, content: editingData[type].content }
+      : { header: editingData[type].header, content: newVal };
+    setEditingData({
+      ...editingData,
+      [type]: updatedVal
+    });
+  };
+
+  // map over every text section in a hackathon's WebsiteData and adds the section name to isEditingObj state
+  return (
+    <>
+      {Object.keys(websiteData).map(type => {
+        return (
+          <Card key={`card${type}`}>
+            <CardHeader>
+              <CardTitle>{websiteData[type].title}</CardTitle>
+              <p>
+                {`Last edited by ${websiteData[type].editor} at ${new Date(
+                  websiteData[type].time
+                ).toLocaleString()}`}
+              </p>
+              <CardButtonContainer>
+                <Button type={EDIT} onClick={() => handleEdit(type)} />
+              </CardButtonContainer>
+            </CardHeader>
+            <CardContent>
+              {isEditingObj[type] ? (
+                <div style={{ padding: '0px 40px 37px 40px' }}>
+                  <StyledHeader>Header</StyledHeader>
+                  <TextBox
+                    defaultValue={editingData[type].header}
+                    onChange={event => handleEditChange(event, type, true)}
+                  />
+                  <StyledHeader>Body</StyledHeader>
+                  <TextBox
+                    defaultValue={editingData[type].content}
+                    resize
+                    onChange={event => handleEditChange(event, type, false)}
+                  />
+                  <div
+                    style={{
+                      marginTop: '27px',
+                      display: 'flex',
+                      float: 'right'
+                    }}
+                  >
+                    <StyledCancel onClick={() => handleCancel(type)}>
+                      <p
+                        style={{
+                          borderBottom: `2px solid ${COLOR.BLACK}`,
+                          margin: '0px'
+                        }}
+                      >
+                        Cancel
+                      </p>
+                    </StyledCancel>
+                    <Button onClick={() => handleSave(type)}>Save</Button>
+                  </div>
+                </div>
+              ) : (
+                <>
```

Comment:
Is <> a shortcut for <React.Fragment> ?

Reply:
i think so LOL, i think the linter did that when i ran it